src/auspex/analysis/fits.py

```diff
@@ -58,3 +58,35 @@ def sinf(x, f, A, phi, y0):
 
 def quadf(x, A, x0, b):
     return A*(x-x0)**2+b
+
+def fit_CR(xpoints, data, cal_type):
```

Comment:
1. doc string please
2. if you make cal_type and [enum](https://docs.python.org/3/library/enum.html) the `cal_type` will be more transparent